hilti/runtime/include/context.h

```diff
@@ -85,20 +85,13 @@ extern Context*& current();
 extern Context* master();
 
 /** Returns the context set for the current hardware thread. */
-inline auto get() {
-    assert(current());
```

Comment:
Suggestion: Rather than completely disabling the null check, how about adding a boolean parameter `allow_missing_context` with default `false`. We'd then set that to true from just those two locations where it can happen for benign reasons.